splash/src/main/res/drawable/splash_gradient.xml

```diff
@@ -0,0 +1,11 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <gradient
+        android:angle="90"
+        android:endColor="#2ea1a2"
+        android:startColor="#1c515e" />
+
+    <corners android:radius="0dp" />
```

Comment:
Isn't `0dp` the default radius? It should look the same if you delete this tag.

Reply:
Removed.